### PR TITLE
Adds a git_hash and git_timestamp to get the commit timestamp

### PR DESCRIPTION
**Pull request by awholegunch**
**Displays these in the meta tags of each page.** This will show which version of the documentation has been used to build the HTML files.

Fix for https://issues.cask.co/browse/CDAP-2539

See the meta tags of any of the pages at: http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT-feature-3.1_add_doc_timestamp/en/index.html

They will appear as:
```
<meta name="git_release" content="3.1.0-SNAPSHOT">
<meta name="git_hash" content="876eed20451495d7095d4875eba00b66c523cc7d">
<meta name="git_timestamp" content="2015-05-27 20:27:48 -0700">
```

### CODE REVIEW

**Comment by duddun** [comment]
LGTM

File: cdap-docs/_common/_themes/cdap/theme.conf [hunk 1]
```diff
@@ -4,9 +4,10 @@ stylesheet = cdap.css
...
 
 [options]
-stickysidebar = true
 manuals = []
 manual_titles = []
 manual_icons = []
+meta_git = []
+stickysidebar = true
 versions = []
 versions_data = ''
```
Comment (anwar6953) [inline cdap-docs/_common/_themes/cdap/theme.conf]: btw can you add a newline to this file

Comment (awholegunch) [inline cdap-docs/_common/_themes/cdap/theme.conf]: done.

**Comment by duddun** [comment]
It looks Ok to me.

**Pull request merged**